### DISCUSSION TITLE
NPM dependency auditing

### DISCUSSION THREAD

**Discussion by croftj** [padloc/padloc#662] [Q&A] [answered]
Hi,

Padloc looks like it might be just what I am looking for!  I will continue playing with it.

Being a bit paranoid, are there any auditing of security issues found in the dependent packages for padloc. Sadly, I am not very knowledgeable of javascript  / npm and don't really have time to learn the magic. I am an just another end user. 

Thank you,
     Joe

**Comment by BrunoBernardino** [comment, marked as answer]
Hey @croftj, thanks for reaching out, here. You can see the automatic dependency security audits performed in https://github.com/padloc/padloc/security and also, if you pull down the repo and install the dependencies (`npm ci`), you'll see a report of potential security threats reported (currently none).

You can also read through the recent third-party security audit Padloc received here: https://padloc.app/blog/security-audit-ros/

**Comment by croftj** [comment]
Thanks Bruno!
